src/OrbitQt/main.cpp

```diff
@@ -84,6 +85,11 @@ void RunUiInstance(const DeploymentConfiguration& deployment_configuration,
 
   std::optional<orbit_qt::TargetConfiguration> target_config;
 
+  ErrorMessageOr<orbit_metrics_uploader::MetricsUploader> metrics_uploader = ErrorMessage("");
+  if (absl::GetFlag(FLAGS_enable_metrics)) {
+    metrics_uploader = orbit_metrics_uploader::MetricsUploader::CreateMetricsUploader();
```

Comment:
Could you add some log messages here? Maybe printing the error message, if there is one and if the creation succeeds maybe a message saying that the metrics uploader has been loaded properly or something like that?

Reply:
Discussed offline: I've added logging in the `main.cpp` (and leave the `ErrorMessageOr` type), and removed logging from MetricsUploader::Create().